src/buildCommon.js

```diff
@@ -139,12 +139,15 @@ const mathDefault = function(
     } else if (type === "textord") {
         const font = symbols[mode][value] && symbols[mode][value].font;
         if (font === "ams") {
+            const fontName = retrieveTextFontName("amsrm", options.fontStyles);
             return makeSymbol(
-                value, "AMS-Regular", mode, options, classes.concat(["amsrm"]));
+                value, fontName, mode, options,
+                classes.concat(options.fontStyles, "amsrm"));
```

Comment:
I feel like we should just pass `fontStyles` to `makeSymbol` so that the caller isn't responsible for calling `retrieveTextFontName` and concatenating classes.